iphone.css

```diff
@@ -0,0 +1,224 @@
+html, body, div, span, applet, object, iframe,
```

Comment:
same comment as above in the ipad.css file. it is unnecessary to re-write all this code from main.css